LLVM.Imports.BitReader.pas

```diff
@@ -9,16 +9,25 @@ interface
   LLVM.Imports.Types;
 
 //LLVMParseBitcode is deprecated, therefore we implement only LLVMParseBitcode2, but as LLVMParseBitcode
-function LLVMParseBitcode(MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcode2';
+function LLVMParseBitcode(MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef; var OutMessage: PLLVMChar): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcode2';
+//* Builds a module from the bitcode in the specified memory buffer, returning a  reference to the module via the OutModule parameter. Returns 0 on success. */
+function LLVMParseBitcode2(MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcode2';
 
 //LLVMParseBitcodeInContext is deprecated, therefore we implement only LLVMParseBitcodeInContext2, but as LLVMParseBitcodeInContext
-function LLVMParseBitcodeInContext(ContextRef: TLLVMContextRef; MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcodeInContext2';
+function LLVMParseBitcodeInContext(ContextRef: TLLVMContextRef; MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef;OutMessage: PLLVMChar): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcodeInContext2';
+// but as LLVMParseBitcodeInContext
+function LLVMParseBitcodeInContext2(ContextRef: TLLVMContextRef; MemBuf: TLLVMMemoryBufferRef; out OutModule: TLLVMModuleRef): TLLVMBool; cdecl; external CLLVMLibrary name 'LLVMParseBitcodeInContext2';
```

Comment:
Same issue as LLVMPareBitcode(2)

Reply:
yes exactly but it remains obsolete so I had eliminated it but then I reinserted it for compatibility.

Reply:
This included, there are still some wrong imports using the same "name" deklaration.

Reply:
Fixed, who knows why they had escaped me. ;);)

Reply:
:)
Btw theoretically cou can remove the "name" declarations now, as declared function and imported function have the same name. But that's nothing i'll force upon this review.

Reply:
yes, but everything had been done with the declaration of names

Reply:
only for the ones where i had changed names ;)

Reply:
.